README.rst

```diff
@@ -4,28 +4,85 @@ eventyay-tickets
 .. image:: https://codecov.io/gh/fossasia/eventyay-tickets/branch/master/graph/badge.svg
    :target: https://codecov.io/gh/pretix/pretix
 
-
 Project status & release cycle
 ------------------------------
 
-Welcome to the eventyay project. The ticketing component of the system provides options for ticket sales and event related items such a Tshirts. Eventyay is developed since 2014. The ticketing component is based on a fork of pretix.
+Welcome to the **Eventyay** project! The ticketing component of the system provides options for **ticket sales and event-related items** such as T-shirts. Eventyay has been in development since **2014** and is based on a fork of **Pretix**.
 
 Support
 -------
 
-This project is free and open source software. Professional support is available to customers of the hosted eventyay service or eventyay enterprise offerings. If you are interested in commercial support, hosting services or supporting this project financially, please go to `eventyay.com`.
+This project is **free and open-source software**. Professional support is available to customers of the **hosted Eventyay service** or **Eventyay enterprise offerings**. If you are interested in commercial support, hosting services, or supporting this project financially, please go to `eventyay.com`.
 
 Contributing
 ------------
+
 Please look through our issues and start contributing.
 
+### Setting Up Eventyay-Tickets Locally
```

Comment:
Please create a "Docker based" section for this.
Docker is not always needed. It is just that the non-Docker setup has been broken that we have to drive contributors to the Docker-based setup.